### PR TITLE
Deal with CI changes (in the compiler this time).

### PR DESCRIPTION
**Pull request by vext01**
The last of these PRs.

Assigned both @ltratt and @ptersilie. Who will get there first? 😃

### CODE REVIEW

File: .buildbot.sh [hunk 1]
```diff
@@ -4,12 +4,18 @@
 
 set -e
 
+case ${CI_TRACER_KIND} in
+    "sw" | "hw" ) true;;
```
Comment (ptersilie) [inline .buildbot.sh:8]: I'm not a ~~bash~~ shell expert, but this syntax looks odd. Are there really two closing brackets `)` but no opening ones?

Comment (vext01) [reply .buildbot.sh:8]: It's the way it is...
https://www.shellscript.sh/case.html

Comment (ptersilie) [reply .buildbot.sh:8]: Urgh!

**Comment by ptersilie** [comment]
> Already running a review

Aww :disappointed:

**Pull request merged**